providers/base/units/stress/jobs.pxu

```diff
@@ -607,19 +607,26 @@ plugin:shell
 category_id: com.canonical.plainbox::stress
 id: stress/s2idle_pm-graph_30
 estimated_duration: 10m
-requires: 
+requires:
+ cpuinfo.type == 'GenuineIntel'
  executable.name == 'sleepgraph'
  sleep.mem_sleep == 's2idle'
 user: root
 _summary: Resume from idle by using Intel pm-graph
 command:
- sleepgraph -m freeze -rtcwake 10 -sync -gzip -multi 30 0 -skiphtml -o "$PLAINBOX_SESSION_SHARE"/s2idle_pm-graph/s2idle-"$(date -d today +%Y-%m-%d-%H%M)"
+ if [ ! -f /proc/driver/nvidia/suspend ]; then
+     sleepgraph -m freeze -rtcwake 60 -sync -gzip -multi 30 30 -skiphtml -o "$PLAINBOX_SESSION_SHARE"/s2idle_pm-graph/s2idle-"$(date -d today +%Y-%m-%d-%H%M)"
+ else
+     echo "The platform has Nvidia driver loaded, which is not capable of running sleepgraph"
+     exit 1
```

Comment:
```suggestion
     echo "The platform has Nvidia driver loaded, which is not capable of running sleepgraph. Exiting..."
```

If you `exit 1`, the job will be marked as failed on all the devices with a Nvidia GPU, which is probably not what you want. Just `echo` something clear for the reviewer and you're good to go.